src/Menu/GetStarted.php

```diff
@@ -65,4 +95,31 @@ protected function register_navigation_pages(): void {
 			]
 		);
 	}
+
+	/**
+	 * Maybe redirect to dashboard.
+	 *
+	 * @return void
+	 */
+	public function maybe_redirect_to_dashboard(): void {
+		if ( $this->wp->wp_doing_ajax() ) {
+			return;
+		}
+
+		if ( ! $this->is_current_wc_admin_page( self::$path ) ) {
+			return;
+		}
+
+		$this->redirect_to_dashboard();
+	}
+
+	/**
+	 * Redirect to start page and exit.
+	 *
+	 * @return void
+	 */
+	private function redirect_to_dashboard(): void {
+		wp_safe_redirect( admin_url( 'admin.php?page=wc-admin&path=' . rawurlencode( Dashboard::$path ) ) );
+		exit;
+	}
```

Comment:
This two functions are 90% equal than the ones defined in the Dashboard redirection. The only thing changing is the referred path.  So maybe we can refactor them as something like:

`redirect_to_path( $path )`